enterprise_gateway/services/processproxies/spark_operator.py

```diff
@@ -22,3 +28,61 @@ def __init__(self, kernel_manager: RemoteKernelManager, proxy_config: dict):
         self.group = "sparkoperator.k8s.io"
         self.version = "v1beta2"
         self.plural = "sparkapplications"
+
+    def get_initial_states(self) -> set:
```

Comment:
These methods should reside on the `CustomResourceProcessProxy` since they provide nothing specific to the Spark Operator. (The help string in `get_container_status` should replace "spark operator" with "custom resource" or "custom resource definition".)